docs/resource/k8s.md

```diff
@@ -91,6 +91,7 @@ containers on your cluster.
 <!-- semconv k8s.pod -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| [`k8s.pod.annotations.<key>`](../attributes-registry/k8s.md) | string | The annotations placed on the Pod, the `<key>` being the annotation name, the value being the annotation value. | `k8s.pod.annotations.kubernetes.io/enforce-mountable-secrets=true`; `k8s.pod.annotations.mycompany.io/arch=x64`; `k8s.pod.annotations.data=` | Opt-In |
 | [`k8s.pod.labels.<key>`](../attributes-registry/k8s.md) | string | The labels placed on the Pod, the `<key>` being the label name, the value being the label value. | `k8s.pod.labels.app=my-app`; `k8s.pod.labels.mycompany.io/arch=x64`; `k8s.pod.labels.data=` | Recommended |
```

Comment:
Maybe a nit, but I wonder if `annotations` and `labels` should be pluralized or not.

There's a similar pattern used by stable HTTP conventions, there the singular is used ([`http.response.header.<key>`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md#common-attributes)).

It would make sense to settle on the singular here too (`k8s.pod.annotation.<key>` and `k8s.pod.label.<key>`), for consistency, and also because each attribute only refers to a single label or annotation.

Reply:
I don't have a strong preference for that so yeap maybe it's a good idea to align `annotations` and `labels` with what we have in `http.response.header.*`.

What @open-telemetry/semconv-k8s-approvers think about this?

Reply:
I think being consistent is important, so sounds like we should have `label` and `annotation`.

Reply:
It makes sense to drop the pluralization given the existing guidance. Once this lands, we'll need to update the k8sattr processor which uses `labels` and `annotations` at present.

Reply:
Thank's folks. Would you mind giving another review on the updated PR?
Happy to help with the implementation's side changes as well :).

Reply:
good catch 👍